src/errors/types.rs

```diff
@@ -541,8 +541,14 @@ impl ErrorType {
     pub fn message_template_json(&self) -> &'static str {
         match self {
             Self::NoneRequired => "Input should be null",
-            Self::ListType => "Input should be a valid array",
-            Self::DataclassType { .. } => "Input should be an object",
+            Self::DictAttributesType => "Input should be a valid object or instance to extract fields from",
```

Comment:
Should we explicitly add all of them, or it's fine to have the fallback?

Reply:
fine to have fallback.

I think this one can be removed add added to the `"Input should be an object"` message.